packages/astro/src/default/pages/index.astro

```diff
@@ -9,11 +10,13 @@ const part = lesson.part && tutorial.parts[lesson.part.id];
 const chapter = lesson.chapter && part?.chapters[lesson.chapter.id];
 
 const slug = [part?.slug, chapter?.slug, lesson.slug].filter(Boolean).join('/');
+const meta = lesson.data?.meta ?? {};
 
 const redirect = joinPaths(import.meta.env.BASE_URL, `/${slug}`);
 ---
 
 <!doctype html>
 <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
 <title>Redirecting to {redirect}</title>
```

Comment:
Or maybe we should pass constructed `lesson.title` to `<MetaTags>`, so that it's only used in meta tags. Title of document/head would still remain the same. 🤔

Reply:
Yes you'd definitely want to have the redirection page to have the same meta as the lesson's page, this is just a side-effect of Astro using client-side redirection to avoid a strong dependence on a server.
Indeed I think the document title can stay the same, it will be visible when navigating on the page but not on social media as far as I understand.

Reply:
Redirect page works now:

<img src="https://github.com/user-attachments/assets/53d603aa-72b7-4f15-9f8c-5b981c023ac4" width="400" />